packages/p2p/src/components/modal-manager/modals/create-ad-modal/ad-component.jsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Button, Text, Icon } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import './ad-component.scss';
+
+const AdComponent = ({ title, text, type }) => {
```

Comment:
Could we rename the component to maybe CreateAdModalCard or something that relates to the component, it feels a little too vague the naming convention... also could you make sure the separate this component into another folder within the create-ad-modal folder, just to keep things clean 👍

Reply:
and add another index.js there as well!